src/layouts/md/Translatathon.tsx

```diff
@@ -162,22 +164,25 @@ export const TranslatathonLayout = ({
     ),
     buttons: [
       {
-        content: "Apply to translate",
-        href: APPLICATION_URL,
+        content: (
+          <PaperformCallToAction content="Apply to translate" variant="solid" />
+        ) as ReactNode,
```

Comment:
this is created a nested button since the ContentHero is already printing a button for this content